rules/prefer-node-remove.js

```diff
@@ -46,34 +36,28 @@ const getArgumentName = arguments_ => {
 
 const create = context => {
 	return {
-		CallExpression(node) {
+		'CallExpression[callee.property.name="removeChild"]'(node) {
 			const {callee} = node;
 
-			if (node.arguments.length === 0 ||
+			if (node.arguments.length !== 1 ||
 				callee.type !== 'MemberExpression' ||
```

Comment:
Let's add this to selector too

Reply:
Wait, is this always true? when [callee.property.name="removeChild"]

Reply:
I've send #509 to fix `prefer-node-append`

Reply:
I can look into these changes as I have time, but if there is refactoring into a common utility (e.g., for setting the selector), it would seem we should look at all affected functions together.